### PR TITLE
Implement realtime audio playback option

### PR DESCRIPTION
**Pull request by OpenSauce04**
This pull request ports the realtime audio option which was recently added to Citra Enhanced, although I have made a notable improvement to how the feature behaves.

The version of the feature which is currently present in Citra Enhanced calculates the audio timescale by using only data from the previous frame. This means that if the emulation framerate is inconsistent, the audio playback will reflect this inconsistency, leading to frequent and very noticeable speedups and slowdowns.

To remedy this, I have tweaked the implementation of this feature when bringing it over to Lime3DS. Instead of only using the data from the previous frame to calculate the audio timescale, a mean average of the last 50 displayed frames is used. This results in significantly smoother playback when the framerate is inconsistent.

This '50 frames' number was the result of personal testing. Values lower than this did not properly address the issue enough and still had speedups and slowdowns in audio playback when framerate was inconsistent, whereas values higher than this did not account for changes in framerate fast enough, which resulted in the same issue when entering large lag spikes. 50 frames of data seems to be the Goldilocks zone for consistent playback

### CODE REVIEW

File: src/android/app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -246,6 +246,8 @@ Se esperan fallos gráficos temporales cuando ésta esté activado.</string>
     <string name="audio_volume">Volumen</string>
     <string name="audio_stretch">Extensión de Audio</string>
     <string name="audio_stretch_description">Extiende el audio para reducir los parones. Cuando se active, la latencia de audio se incrementará y reducirá un poco el rendimiento.</string>
+    <string name="realtime_audio">Activar audio en tiempo real</string>
+    <string name="realtime_audio_description">Simula el audio de la Nintendo 3DS para HLE, corrigiendo el retraso del audio cuando la velocidad es inferior al 100%. Podría causar problemas de desincronización del audio</string>
```
Comment (Gamer64ytb) [inline src/android/app/src/main/res/values-es/strings.xml]: Here an updated text for spanish:

Ajusta la velocidad de reproducción de audio para compensar las caídas en la velocidad de emulación de cuadros. Esto significa que el audio se reproducirá a velocidad completa incluso cuando la velocidad de cuadros del juego sea baja. Puede causar problemas de desincronización de audio.

**Review by ayrudev — APPROVED** [review]
tldr; it works and the code looks great

just now i've been able to test it, it's working and it works great, i've achieved to slow down my game by cranking up the settings, and the audio didn't crack as it did with the option turned off, the code looks great, so, LGTM
